.gitignore

```diff
@@ -15,3 +15,7 @@ Arelle.egg-info
 *.iml
 .idea
 /venv/
+scripts/conformance_suites/*.txt
+scripts/conformance_suites/*.csv
+scripts/conformance_suites/*.xlsx
+tests/resources/conformance_suites/*.zip
```

Comment:
The UTR suite has nested directories and an XML file, so how about just
`tests/resources/conformance_suites/`